src/Buildalyzer/AnalyzerResult.cs

```diff
@@ -176,16 +176,24 @@ internal void ProcessProject(PropertiesAndItems propertiesAndItems)
 
         internal void ProcessCscCommandLine(string commandLine, bool coreCompile)
         {
-            // Some projects can have multiple Csc calls (see #92) so if this is the one inside CoreCompile use it, otherwise use the first
-            if (string.IsNullOrWhiteSpace(commandLine) || (_cscCommandLineArguments != null && !coreCompile))
+            if (string.IsNullOrWhiteSpace(commandLine))
             {
                 return;
             }
+
             ProcessedCommandLine cmd = ProcessCscCommandLine(commandLine);
-            _command = cmd.Command;
-            _compilerFilePath = cmd.FileName;
-            _compilerArguments = cmd.Arguments.ToArray();
-            _cscCommandLineArguments = cmd.ProcessedArguments;
+
+            // Some projects can have multiple Csc calls (see #92) so if this is the one inside CoreCompile use it, otherwise use the first
+            if (coreCompile)
+            {
+                _command = cmd.Command;

```

Comment:
Why would you set these properties only sometimes? I would expect to get the compiler command-line of any type of project from these properties.

Reply:
Because I took out the condition above that if "_cscCommandLineArguments != null && !coreCompile" then the function would exit early. 
However I have a slight mismatch in the logic, so I'll adjust

when _cscCommandLineArguments != null, !coreCompile, before it exited early, now command etc not set
when _cscCommandLineArguments == null, !coreCompile,  before set command etc, now command etc not set
when _cscCommandLineArguments != null, coreCompile, before set command etc, now set command etc 
when _cscCommandLineArguments == null, coreCompile, before set command etc, now set command etc

Reply:
The point I am trying to make is that I would expect the externally visible properties `CompilerCommand`, `CompilerFilePath` and `CompilerArguments` of `IAnalyzerResult` to be in sync with the contents of the internal `_cscCommandLineArguments` list. Is there any reason why this is not possible?